packages/rocketchat-ui-account/client/accountPreferences.js

```diff
@@ -89,6 +92,9 @@ Template.accountPreferences.helpers({
 	},
 	notificationsSoundVolume() {
 		return RocketChat.getUserPreference(Meteor.user(), 'notificationsSoundVolume');
+	},
+	allowNotificationsChange() {
```

Comment:
Same with this one.  Change to include email in the name so its more intuitive.  Some reason `emailNotificationsAllowed`  I think makes sense as to what it does